### PR TITLE
Update SearchBar README

### PR DESCRIPTION
**Pull request by paul0vinicius**
#### What is the purpose of this pull request?

As the title says.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SearchBar/README.md [hunk 1]
```diff
@@ -1,24 +1,57 @@
...
-Search Bar is a canonical component that any VTEX app can import.
 
-To import it into your code: 
-```js
-import SearchBar from 'vtex.store-components/SearchBar'
-```
+## Description
+`Search Bar` is a VTEX Component that shows a search bar with autocomplete options displaying the matching products as well. This component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
```
Comment (klzns) [inline react/components/SearchBar/README.md]: Following changes made in minicart

```suggestion
:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
```

File: react/components/SearchBar/README.md [hunk 2]
```diff
@@ -1,24 +1,57 @@
 # Search Bar
-Search Bar is a canonical component that any VTEX app can import.
 
-To import it into your code: 
-```js
-import SearchBar from 'vtex.store-components/SearchBar'
-```
+## Description
+`Search Bar` is a VTEX Component that shows a search bar with autocomplete options displaying the matching products as well. This component can be imported and used by any VTEX App.
```
Comment (theomoura) [inline react/components/SearchBar/README.md]:
```suggestion
`Search Bar` is a VTEX Component that shows a search bar with autocomplete options and displays the matching products as well. This component can be imported and used by any VTEX App.
```

File: react/components/SearchBar/README.md [hunk 3]
```diff
@@ -1,24 +1,57 @@
...
 
-| Prop name          | Type      | Description                                                          |
-| ------------------ | --------- | -------------------------------------------------------------------- |
-| `placeholder`      | `String`  | Displays a placeholder on the search bar                             |
-| `emptyPlaceholder` | `String`  | Shows a placeholder when the ResultList hasn't results to displayed  |
+For now this block does not have any required or optional blocks.
+
+#### Configuration
+
+Through the Storefront, you can change the `SearchBar`'s behavior and interface. However, you also can make in your theme app, as Store theme does. However, you also can make in your theme app, as Store theme does.
```
Comment (theomoura) [inline react/components/SearchBar/README.md]:
```suggestion
Through the Storefront, you can change the `SearchBar`'s behavior and interface. However, you also can make in your theme app, as Store theme does.
```

File: react/components/SearchBar/README.md [hunk 4]
```diff
@@ -1,24 +1,57 @@
...
+
+| Prop name          | Type      | Description                                                          | Default value
+| ------------------ | --------- | -------------------------------------------------------------------- |------------- |
+| `placeholder`      | `String!`  | Placeholder to be used on the input                             | - |
+| `emptyPlaceholder` | `String!`  | Shows a placeholder when the ResultList hasn't results to displayed  | - |
+| `compactMode` | `Boolean`  | Indentify when use the compact version of the component  | - |
```
Comment (theomoura) [inline react/components/SearchBar/README.md]:
```suggestion
| `compactMode` | `Boolean`  | Identify when use the compact version of the component  | - |
```

**Pull request merged**